### ISSUE TITLE
Support for tensorflow 1.14?

### ISSUE THREAD

**Issue by panfeng-hover** [aws/sagemaker-tensorflow-container#226]
Hello, I am wondering when will the docker image support for tensorflow 1.14 available? Thanks!

`The image '520713654638.dkr.ecr.us-east-1.amazonaws.com/sagemaker-tensorflow:1.13-gpu-py3' does not exist.`

**Comment by knakad** [comment]
Actually, we're actively currently working on releasing 1.14! We'll follow up when it's ready =)

Out of curiosity, what new feature are you aiming for with 1.14?

You also linked an error regarding 1.13. Was that purposeful?

**Comment by panfeng-hover** [comment]
Oh, I think I actually need the tf 1.14 for `sagemaker-tensorflow-serving-container`. The above error is due to `TensorFlowModel` not support `py3`.

@ChoiByungWook gives a detail explanation in a similar issue. https://github.com/aws/sagemaker-python-sdk/issues/912

The reason I need tf 1.14 is basically I need to use `mask-rcnn` model, currently in `tensorflow/models` repo, the `mask-rcnn` not support multiple gpu training, which is quite slow. So I decide to switch to use the one in `tensorflow/tpu`, which support a lot larger batch size, and it requires tf 1.14 for official support for `mask-rcnn`.

Since currently, SageMaker doesn't provide `mask-rcnn` model for training, I generally trained the model somewhere firstly and then deploy it on SageMaker endpoint.

**Comment by icywang86rui** [comment]
Like @knakad mentioned we are currently working on 1.14 release and it will be available in the near future. Sorry that you are blocked for now. Closing the issue for now. Feel free to reopen if you have any further questions.

**Comment by Juliopr** [comment]
so. did the 1.14 release happened? or its only python 2.7 with tensroflow 1.12.

**Comment by laurenyu** [comment]
@panfeng-hover @ks233ever @Juliopr @Rhuax TF 1.14 support has been released. You can find the full list of supported versions at https://github.com/aws/sagemaker-python-sdk#tensorflow-sagemaker-estimators

**Comment by ks233ever** [comment]
Has anyone deployed using 1.14 yet? I'm getting an error that the image doesn't exist
